frontend/packages/process-editor/src/components/ConfigPanel/ConfigPanel.tsx

```diff
@@ -25,35 +17,68 @@ export interface ConfigPanelProps {
 export const ConfigPanel = ({
   existingCustomReceiptName,
   onUpdateLayoutSet,
-}: ConfigPanelProps): JSX.Element => {
+}: ConfigPanelProps): React.ReactElement => {
+  return (
+    <ConfigSurface>
+      <ConfigPanelContent
+        existingCustomReceiptName={existingCustomReceiptName}
+        onUpdateLayoutSet={onUpdateLayoutSet}
+      />
+    </ConfigSurface>
+  );
+};
+
+type ConfigPanelContentProps = ConfigPanelProps;
+const ConfigPanelContent = ({
+  existingCustomReceiptName,
+  onUpdateLayoutSet,
+}: ConfigPanelContentProps): React.ReactElement => {
   const { t } = useTranslation();
   const { bpmnDetails } = useBpmnContext();
-  const [showEndEventConfig] = React.useState<boolean>(shouldDisplayFeature('customizeEndEvent'));
-
-  const displayContent = () => {
-    if (bpmnDetails === null || bpmnDetails.type === BpmnTypeEnum.Process) {
-      return (
-        <Paragraph className={classes.configPanelParagraph} size='small'>
-          {t('process_editor.configuration_panel_no_task')}
-        </Paragraph>
-      );
-    } else if (showEndEventConfig && bpmnDetails.type === BpmnTypeEnum.EndEvent) {
-      return (
-        <ConfigEndEvent
-          existingCustomReceiptName={existingCustomReceiptName}
-          onUpdateLayoutSet={onUpdateLayoutSet}
-        />
-      );
-    } else if (bpmnDetails.type === BpmnTypeEnum.Task) {
-      return <ConfigContent />;
-    } else {
-      return (
-        <Paragraph className={classes.configPanelParagraph} size='small'>
-          {t('process_editor.configuration_panel_element_not_supported')}
-        </Paragraph>
-      );
-    }
-  };
-
-  return <div className={classes.configPanel}>{displayContent()}</div>;
+
+  const noTaskSelected = bpmnDetails === null || bpmnDetails.type === BpmnTypeEnum.Process;
+  if (noTaskSelected) {
+    return (
+      <BpmnAlert
+        title={t('process_editor.configuration_panel_no_task_title')}
+        message={t('process_editor.configuration_panel_no_task_message')}
+      />
+    );
+  }
+
+  if (shouldDisplayFeature('customizeEndEvent') && bpmnDetails.type === BpmnTypeEnum.EndEvent) {
```

Comment:
Should we have a similar const for this case as for the others? Only for consistency

Reply:
Sure! 😄